### PR TITLE
[FSDE-31351] Add functional example of closable alert in doc

### PR DESCRIPTION
**Pull request by mtadejalumen**
https://ctl.atlassian.net/browse/FSDE-31351

### CODE REVIEW

File: src/website/views/components/alert/_examples.pug [hunk 1]
```diff
@@ -552,6 +552,41 @@ p.-text
...
+    chi-alert#interactive-alert(icon='circle-info', closable, mutable).-m--2
+      | This is a base alert
+  .example-tabs.-pl--2
+    ul.chi-tabs#example-interactive
+      li.-active
+        a(href=`#webcomponent-portal-interactive`) Web Component
+      li
+        a(href=`#html-portal-interactive`) HTML Blueprint
+  .chi-tabs-panel.-active#webcomponent-portal-interactive
+    .chi-tab__description
```
Comment (jllr) [inline src/website/views/components/alert/_examples.pug:570]:
```suggestion
```

File: src/website/views/components/alert/_examples.pug [hunk 2]
```diff
@@ -552,6 +552,41 @@ p.-text
           </button>
         </div>
 
+h4 Interactive
+p.-text
+.example.-mb--3
+  .-p--3#interactive-wrapper
+    chi-button#alert-trigger-button(variant='outline')
+      | Click me to show the alert
+    chi-alert#interactive-alert(icon='circle-info', closable, mutable).-m--2
```
Comment (jllr) [inline src/website/views/components/alert/_examples.pug:561]:
```suggestion
    chi-alert.-mt--2#interactive-alert(title='Warning', color='warning', icon='warning', closable)
```

File: src/website/views/components/alert/_examples.pug [hunk 3]
```diff
@@ -552,6 +552,41 @@ p.-text
...
+      | This is a base alert
```
Comment (jllr) [inline src/website/views/components/alert/_examples.pug:562]:
```suggestion
      | This is a closable warning alert
```

File: src/website/views/components/alert/_examples.pug [hunk 4]
```diff
@@ -552,6 +552,41 @@ p.-text
           </button>
         </div>
 
+h4 Interactive
```
Comment (jllr) [inline src/website/views/components/alert/_examples.pug:555]:
```suggestion
h4 Closable alert interaction
```

**Review by jllr — CHANGES_REQUESTED** [review]
@mtadejalumen please change the alert in the example to warning with title. 
Also update code snippets to reflect the changes.
Screen Shot 2023-06-06 at 4 27 19 PM

File: src/website/views/components/alert/_examples.pug [hunk 5]
```diff
@@ -1699,3 +1734,16 @@ script.
     function() {
       chi.tab(document.querySelectorAll('.chi-tabs-panel .chi-tabs'));
     });
+    setupInteractiveAlert();
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/alert/_examples.pug:1737]: I would change a little bit the `script` part:

image

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

File: src/website/views/components/alert/_examples.pug [hunk 6]
```diff
@@ -552,6 +552,40 @@ p.-text
           </button>
         </div>
 
+h4 Closable alert interaction
+p.-text
+.example.-mb--3
+  .-p--3#interactive-wrapper
+    chi-button#alert-trigger-button(variant='outline')
```
Comment (jllr) [inline src/website/views/components/alert/_examples.pug:559]:
```suggestion
    chi-button#alert-trigger-button(color='primary')
```

**Review by jllr — CHANGES_REQUESTED** [review]
@mtadejalumen code snippet should include the button that trigger the action. Similar as it is in the [modal](https://lib.lumen.com/chi/5.33.0/components/modal/?theme=Portal#base), [drawer](https://lib.lumen.com/chi/5.33.0/components/modal/?theme=Portal#base)

File: src/website/views/components/alert/_examples.pug [hunk 7]
```diff
@@ -552,6 +552,94 @@ p.-text
...
+    :code(lang="html")
+      <!-- Wrapper -->
+      <div id="interactive-wrapper">
+        <!-- Trigger -->
+        <chi-button id="alert-trigger-button" color="primary">Click me to show the alert</chi-button>
+
+        <!-- Alert -->
+        <chi-alert id="interactive-alert" icon="circle-info" closable mutable>
+          This is a closable warning alert
+        </chi-alert>
```
Comment (jllr) [inline src/website/views/components/alert/_examples.pug:582]: @mtadejalumen code snippets are not refelecting warning alert in web component and html blueprint. Please update

File: src/website/views/components/alert/_examples.pug [hunk 8]
```diff
@@ -552,6 +552,94 @@ p.-text
...
+        const interactiveAlertTrigger = document.getElementById('alert-trigger-button');
+
+        interactiveAlert.parentNode.removeChild(interactiveAlert);
+
+        interactiveAlertTrigger.addEventListener('chiClick', () => {
+          if (interactiveAlertWrapper.children.length === 1) {
+            interactiveAlertWrapper.appendChild(interactiveAlert.cloneNode(true));
+          }
+        })
+      </script>
```
Comment (jllr) [inline src/website/views/components/alert/_examples.pug:598]: @mtadejalumen This logic is only needed for the example that we have in documentation. For developers the only needed logic is one button that triggers an alert. Please remove unnecessary code.

**Review by jllr — CHANGES_REQUESTED** [review]

**Review by jllr — APPROVED** [review]

**Review by josecarlosgonzalezv — APPROVED** [review]

**Pull request merged**